cmd/utils/flags.go

```diff
@@ -421,6 +421,16 @@ var (
 		Name:  "miner.extradata",
 		Usage: "Block extra data set by the miner (default = client version)",
 	}
+	CeloFeeCurrencyDefault = cli.Float64Flag{
+		Name:  "celo.feecurrency.default",
+		Usage: "Default fraction of block gas limit available for TXs paid with a whitelisted currency",
```

Comment:
Just a nit, but I would change it to "whitelisted alternative currency"
I think that feeCurrency and whitelisted means the same for us, but from someone new reading the help, maybe is not clear if Celo is or not part of that batch